src/components/ChromeRoute/ChromeRoute.tsx

```diff
@@ -32,21 +31,13 @@ const ChromeRoute = memo(
     const { setActiveHelpTopicByName } = useContext(HelpTopicContext);
     const { user } = useContext(ChromeAuthContext);
     const gatewayError = useSelector(({ chrome: { gatewayError } }: ReduxState) => gatewayError);
-    const chromeModules = useAtomValue(chromeModulesAtom);
-    const defaultTitle = chromeModules?.[scope]?.defaultDocumentTitle || scope;
 
-    const [activeModule, setActiveModule] = useAtom(activeModuleAtom);
+    const [, setActiveModule] = useAtom(activeModuleAtom);
 
     useEffect(() => {
       batch(() => {
         // Only trigger update on a first application render before any active module has been selected
         // should be triggered only once per session
-        if (!activeModule) {
-          /**
-           * Default document title update. If application won't update its title chrome sets a title using module config
-           */
-          dispatch(updateDocumentTitle(defaultTitle || 'Hybrid Cloud Console'));
```

Comment:
Because I don't remember, please make sure the `updateDocumentTitle` action is used in other places. If its not, remove it completely from the codebase.

Reply:
`updateDocumentTitle` is being used in a few other places (`AllServices.tsx`, `FavoritedServices.tsx`, `create-chrome.ts`), so I left it in place.